### ISSUE TITLE
Upgrading to 2.6

### ISSUE THREAD

**Issue by gafk** [SEPIA-Framework/sepia-docs#124]
I am currently running 2.5.1 via docker on my Synology NAS and I have set up a raspberry pi client to connect to it.
All works well and I have done quite some modifications, e.g. a lot of custom commands via the teach ui and tweaked some audio settings on the pi (I am using a Jabra Speak 510 via USB).

What would be the best way to upgrade to the latest release? Start from scratch and loose all custom settings? Or is there a better way?

**Comment by fquirin** [comment]
Sounds like a nice setup :-)

For the server: Just use the `update-sepia.sh` script ([a bit more info](https://github.com/SEPIA-Framework/sepia-docs/wiki/Update-SEPIA)). Just to be 100% sure I'd recommend to make a backup of the whole SEPIA folder as well, although the script should to the same thing. It will restore your plugins, commands and settings after the update. Check the script output carefully for more info and errors. In the worst case you can restore everything using your backup and try again ^^.

For the client it is a bit more tricky because I've modified run scripts as well, but since your setup seems to work you might not need those changes. I'd try the following:
- Backup the folders 'clexi' and 'sepia-client' just to be safe
- Go to your 'sepia-client' folder and download the new update script: `wget https://raw.githubusercontent.com/SEPIA-Framework/sepia-installation-and-setup/master/sepia-client-installation/rpi/sepia-client/update.sh` (you can just copy it to the 'sepia-client' folder manually as well).
- Shutdown the client and then run `bash update.sh`
- Compare your original and new `settings.js` in `~/clexi/www/sepia/settings`. The format has been changed a bit. Unfortunately you have to transfer your old settings manually, but most of it should be self-explaining (I hope :sweat_smile: )

Settings for your USB mic etc. should remain unchanged, but the new client has a media-devices settings section. If 'default' worked for you so far then it should still be the case after the update. If not I'd recommend to connect a display to the client and switch to 'pseudo-display' mode (if you haven't already) to play with the new settings. There is a new "export settings" button on settings page 2 (below user-name) to generate the headless-client `settings.js` content that can then be copied over to your file. Maybe it makes sense to check-out the new client version at https://sepia-framework.github.io/app/index.html first :-)

Let me know if everything worked out :crossed_fingers: :smiley:

**Comment by fquirin** [comment]
> Only the Android APK did not install properly, but I guess this is because I have the AppStore version installed and I tried to update with the APK

Yes, since they have the same app ID and signature they cannot coexist ^^ (actually its exactly the same APK that I upload to the Play Store). Beta branch in the Play Store was finally accepted today by Google, main brach should follow soon :-)

> Thank you so much for your excellent work, I really enjoy this project a lot and we are using it every day. I might make a short video demo of my setup to promote this project if I find the time. Hope that is ok?

I'm really happy to hear that! :heart_eyes: Of cause you can make any video you like and any promotion would be very welcome :-)
